README.md

```diff
@@ -123,6 +123,8 @@ The following table lists all valid connection properties:
 | USER                       | Yes      | If AUTHENTICATOR is set to `externalbrowser` or the URL for native SSO through Okta, set this to the login name for your identity provider (IdP).     |
 | WAREHOUSE                  | No       |                                                                               |
 | CONNECTION_TIMEOUT         | No       | Total timeout in seconds when connecting to Snowflake. The default is 120 seconds |
+| BROWSER_RESPONSE_TIMEOUT   | No       | Number to seconds to wait for authentication in an external browser (default: 120). |
```

Comment:
It might be worth moving this after `AUTHENTICATOR`, since this seems to apply to `AUTHENTICATOR=externalbrowser`.

Reply:
Done. (Though I originally thought it was more like timeout :)